### ISSUE TITLE
connection.dropCollection()

### ISSUE THREAD

**Issue by AV25242** [couchbaselabs/node-ottoman#306]
connection.dropCollection()
**Parameters**
collection «string» The collection to delete
[callback] «Function»
**Returns:**
«Promise»
Helper for dropCollection(). Will delete the given collection, including all documents and indexes.

**Comment by httpJunkie** [comment]
This ticket has unresolved comments and blocked by:
https://github.com/bwgjoseph/mongoose-vs-ottoman/issues/4

**Comment by AV25242** [comment]
Even though the issue on the customer side is still open, the latest comments indicates that these issues can be closed. We may open a new issue as needed
